daeun/lv1/기사단원의-무기.js

```diff
@@ -0,0 +1,14 @@
+function solution(number, limit, power) {
+    let answer = 0;
+
+    for (let i = 1; i <= number; i++) {
+        let count = 1;
+        for (let j = 1; j <= i / 2; j++) {
+            if (i % j === 0) {
+                count++;
+            }
+        }
+        count > limit ? answer += power : answer += count;
+    }
+    return answer
+}
```

Comment:
```suggestion
function solution(number, limit, power) {
    let answer = 0;

    for (let i = 1; i <= number; i++) {
        let count = 1;
        for (let j = 1; j <= i / 2; j++) {
            if (i % j === 0) {
                count++;
            }
        }
        count > limit ? answer += power : answer += count;
    }
    return answer
}
```
삼항연산자를 활용해서 더 간단하게 코드를 짤 수 있네요! 대운님 풀이 잘 보고 갑니다~!